src/js/angular/ttyg/directives/chat-panel.directive.js

```diff
@@ -193,8 +200,11 @@ function ChatPanelComponent(toastr, $translate, TTYGContextService) {
                 });
             };
 
-            const init = () => {
+            const init = (chat) => {
```

Comment:
why you added these changes here? Better create some reset method or something, because this is confusing now